lib/helpers/deprecator.rb

```diff
@@ -6,11 +6,16 @@
 # rubocop:disable Style/ClassVars
 module Faker
   module Deprecator
+    @@skip_warning = false
+
     def self.included(base)
       extension = Module.new do
         def const_missing(missing_const_name)
           if class_variable_defined?(:@@_deprecated_constants) && (replacement = class_variable_get(:@@_deprecated_constants)[missing_const_name.to_s])
-            $stdout.puts("DEPRECATION WARNING: #{name}::#{replacement[:old_generator]} is deprecated. Use #{replacement[:new_constant]} instead.")
+            unless Faker::Deprecator.skip_warning?
```

Comment:
```suggestion
            if !Faker::Deprecator.skip_warning?
```

There's nothing wrong here with `unless`, it's just that if not reads better.

Reply:
It seems we are preferring unless over `if` with rubocop. Do you want me to change that as well?

Reply:
I am using unless for now, we may want to enforce `if !` with rubocop before this change first.